bootstrap/http/src/main/kotlin/com/studentcenter/weave/bootstrap/meetingTeam/controller/MeetingTeamRestController.kt

```diff
@@ -147,6 +142,18 @@ class MeetingTeamRestController(
         )
     }
 
+    override fun createMeetingTeamInvitation(meetingTeamId: UUID): MeetingTeamCreateInvitationResponse {
+        return MeetingTeamCreateInvitationUseCase.Command(
+            meetingTeamId = meetingTeamId
+        ).let {
+            meetingTeamCreateInvitationUseCase.invoke(it)
+        }.let {
+            MeetingTeamCreateInvitationResponse(
+                teamId = it.teamId,
+                invitationCode = it.invitationCode
+            )
+        }
```

Comment:
p4) 동일한 scope funciton을 여러 쓰는건 가독성이 좋진 않을 것 같아요!
```suggestion
        return meetingTeamCreateInvitationUseCase.invoke(
        	MeetingTeamCreateInvitationUseCase.Command(meetingTeamId = meetingTeamId)
        ).let {
            MeetingTeamCreateInvitationResponse(
                teamId = it.teamId,
                invitationCode = it.invitationCode
            )
        }
```